.github/workflows/release_mac.yml

```diff
@@ -0,0 +1,53 @@
+name: MacRelease
+
+on:
+  push:
+    branches:
+      - rel-*
+  # test this PR for now, remove it before the merge
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: macos-10.15
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        architecture: ['x64']
+    steps:
+    - uses: actions/checkout@v2
+    - name: Checkout submodules
+      shell: bash
+      run: |
+        auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+        git submodule sync --recursive
+        git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: ${{ matrix.architecture }}
+    - name: Install python dependencies
+      run: |
+        python -m pip install --upgrade setuptools
+        pip install pytest==5.4.3 numpy wheel nbval
+        brew update
+        brew install protobuf
+        pip install protobuf==3.11.3
```

Comment:
Unlike onnx/wheel-builder, Installing protobuf 3.11.3 from brew on Mac 10.15 will cause the following issue when using the latest protobuf 3.14.0 on Mac:
```
dlopen(/Users/x/.pyenv/versions/3.7.8/lib/python3.7/site-packages/onnx/onnx_cpp2py_export.cpython-37m-darwin.so, 2): Library not loaded: /usr/local/opt/protobuf/lib/libprotobuf-lite.22.dylib

  Referenced from: /Users/x/.pyenv/versions/3.7.8/lib/python3.7/site-packages/onnx/onnx_cpp2py_export.cpython-37m-darwin.so

  Reason: image not found
```  
Still investigating.

Reply:
Solved by installing static protobuf.